examples/investment_analysis/main.py

```diff
@@ -1,27 +1,26 @@
 from crewai import Agent, Task, Crew, Process
-from composio_langchain import ComposioToolSet, Action, App
-from langchain_google_genai import ChatGoogleGenerativeAI
+from composio_langchain import ComposioToolSet, App
+from langchain_openai import ChatOpenAI
 import os
+import dotenv
 
-# Environment Setup
-os.environ["SERPAPI_API_KEY"] = os.getenv("SERPAPI_API_KEY")
+# Load environment variables
+dotenv.load_dotenv()
 
-# Initialize the language model
-llm = ChatGoogleGenerativeAI(
-    model="gemini-pro", verbose=True, temperature=0.9, google_api_key=os.getenv("GOOGLE_API_KEY")
-)
+# Initialize the language model with OpenAI API key and model name
+llm = ChatOpenAI(openai_api_key=os.getenv("OPENAI_API_KEY", "default_key"), model_name="gpt-4o")
 
-# Define tools for the agents
-composiotoolset = ComposioToolSet()
-tools = composiotoolset.get_actions(actions=[Action.SERPAPI_SEARCH])
+# Setup tools using ComposioToolSet
+composio_toolset = ComposioToolSet()
+tools = composio_toolset.get_tools(apps=[App.SERPAPI])
```

Comment:
Add comments specifically on this line, so it's more obvious this is how we imported tools using composio.